website/static/js/pages/render-nodes.js

```diff
@@ -1,9 +1,15 @@
 'use strict';
 
 var $osf = require('js/osfHelpers');
+var $ = require('jquery');
 
 // model for components, due to simplicity did not create a new file
 var ComponentControl = {};
 
 // binds to component scope in render_nodes.mako
-$osf.applyBindings(ComponentControl, '#componentScope');
+$(document).ready(function() {
+    $('.componentScope').each(function(i) {
+       $(this).attr('id', 'componentScope' + i);
```

Comment:
Required for applyBindings cssSelector, line 464. If I pass the class name, it needs an element ID or jQuery will fail, trying to find something with '# .selector'. If I pass in the DOM element, jQuery fails with "can't find '[object Object] .selector'" or something like that.